### PR TITLE
water hero desktop conversion

### PR DESCRIPTION
**Pull request by bazmurphy**
this pull request revises the water hero, and adds a desktop conversion

### CODE REVIEW

File: 03-water/water-hero/water-hero.css [hunk 1]
```diff
@@ -1,49 +1,130 @@
...
+  row-gap: var(--subcontainer-row-gap-mobile);
+  padding: var(--subcontainer-padding-mobile);
+  background-color: var(--level2-background-color);
+}
+
+@media (min-width: 600px) {
+  .water-hero-subcontainer {
+    grid-template-columns: repeat(2, 1fr);
+    /* grid-template-columns: 1.5fr 2.5fr; */
+    /* ^EXPERIMENTAL */
```
Comment (bazmurphy) [inline 03-water/water-hero/water-hero.css:41]: this is a comment, interesting

File: 03-water/water-hero/water-hero.css [hunk 2]
```diff
@@ -1,49 +1,130 @@
...
+  font-weight: var(--subtitle-font-weight-mobile);
+}
+
+@media (min-width: 600px) {
+  .water-hero-subtitle {
+    text-align: left;
+    font-size: var(--subtitle-font-size-desktop);
+    font-weight: var(--subtitle-font-weight-desktop);
+  }
 }
```
Comment (BoshraM) [inline 03-water/water-hero/water-hero.css:117]: do you think as paragraph container has been added , would be better to have it here for later on if we need to apply css on that.

**Review by BoshraM — APPROVED** [review]

File: 03-water/water-hero/water-hero.html [hunk 3]
```diff
@@ -11,12 +11,23 @@
...
+    
+    <div class="water-hero-subcontainer">
+    
+      <div class="water-hero-image-container">
+        <img class="water-hero-image" src="../../00-overall/05-images/03-water/water-hero-money-waterdrop.png" alt="">
+      </div>
+    
+      <div class="water-hero-copy-container">
+        <h1 class="water-hero-title">Saving water saves Money</h1>
+        <h2 class="water-hero-subtitle">Make a plan to <span class="bold">lessen how much you pay</span> for water</h2>
```
[the remark this answers is not in the document]
Comment (bazmurphy) [reply 03-water/water-hero/water-hero.html:23]: i think the title and subtitle are be okay when they are inside of copy-cointainer...
we only need to nest a title if its further up the hierarchy in a bigger grid like on other components where it sits before other subsections etc.

File: 03-water/water-hero/water-hero.css [hunk 4]
```diff
@@ -1,49 +1,130 @@
...
+  font-weight: var(--paragraph-font-weight-mobile);
+}
+
+@media (min-width: 600px) {
+  .water-hero-paragraph {
+    text-align: left;
+    font-size: var(--paragraph-font-size-desktop);
+    font-weight: var(--paragraph-font-weight-desktop);
+  }
 }
```
Comment (BoshraM) [inline 03-water/water-hero/water-hero.css:130]: good job, overall its good.but images when mediaquery applyes at the start shows smaller than ocntainer wich we shoul come up with solution. for example line 40 you have add a new template which is can control layut

**Pull request merged**